### PR TITLE
added gym website template

### LINKED ISSUE #3594: [Feature Request]: Add a landing page of fitness
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

I would like to add  a landing page of fitness which is made up of html css  

### Use Case

Developers can take inspiration for UI

### Benefits

_No response_

### Add ScreenShots

Screenshot 2024-07-19 180931

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Ridhima10**
Hello PA

Fix #3594 

Screenshot 2024-07-28 170703

### CODE REVIEW

File: src/data/userData.tsx [hunk 1]
```diff
@@ -336,6 +336,19 @@ export const Users: User[] = [
...
   },
+  // Crossfits GYM Template
+
+  {
+    title: "Crossfits",
+    description:
+      "Crossfits is a responsive landing page for GYM and Fitness.",
+    preview: require("./showcase/crossfits.png"),
+    website:
+      "https://github.com/Ridhima10/Crossfits",
```
Comment (Ajay-Dhangar) [inline src/data/userData.tsx:347]: replace `website:
      "https://github.com/Ridhima10/Crossfits",` with `website:
      "https://ridhima10.github.io/Crossfits/",`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by Ridhima10** [comment]
@Ajay-Dhangar  its done. kindly merge it

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**